app/main.py

```diff
@@ -2,0 +1,20 @@
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(visitor=friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            masks += 1
+    if masks > 0:
+        return f"Friends should buy {masks} masks"
```

Comment:
The comparison of `masks` to `0` is not needed here. You can simply check if `masks` is truthy. Here is how you can improve it:
python
if masks:
    return f"Friends should buy {masks} masks"
```
This makes the code more pythonic and efficient.